scripts/update_localization.rb

```diff
@@ -73,6 +76,24 @@
                       "messages#{@options[:pot_suffix]}"
     end
   end
+=end
+  # Remove trailing space
+  info "Removing trailing space..."
+
+  LOCALES.each do |locale|
+    target = locale + @options[:po_suffix]
+    content = File.read(target, :encoding => "utf-8")
+    trimmed = ""
+    content.split("\n").each do |line|
```

Comment:
I think there is an inbuilt method to split things into lines. There's actually a way to read files line by line which is what I've preferred to use. So you could have code like:
```
for each line in file{|line|
stripped = line.strip
if stripped != line
  changes = true
end
lines.append stripped
}

if changes
  write each line to the file
end

```